artview/components/menu.py

```diff
@@ -361,7 +361,7 @@ def _openfile(self):
      
         # Read the data from file
         try:
-            radar = pyart.io.read(self.filename)
+            radar = pyart.io.read(self.filename, delay_field_loading=True)
```

Comment:
We need to make sure we are smart in how we handle errors here. Some file types do not support the 'delay_field_loading' parameter. Does PyART handle this internally and just return all fields if it is not used?